components/kflogin/src/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
```

Comment:
This looks like the stock React CSS file, you probably aren't using anything here (.App is used in App.js but it's not doing anything). Can we remove it?

Reply:
We need ```text-align: center;```
Maybe leave this file for future style edit?